decidim-core/config/locales/en.yml

```diff
@@ -170,6 +170,10 @@ en:
       nesting_too_deep: can't be inside of a subcategory
   layouts:
     decidim:
+      cookie_warning:
+        description_html: This site uses cookies. By continuing to browse the site, you are agreeing to our use of cookies. Find out more about it %{link}.
```

Comment:
Not sure if this is right, if it is tho:

`you agree to something by doing something`

sounds a lot better than

`you are agreeing to something by doing something`